app/konnect/gateway-manager/plugins/index.md

```diff
@@ -0,0 +1,105 @@
+
+In both cases, plugins are managed within a specific control plane. If you need a plugin to 
+apply to multiple control planes, configure a plugin instance separately for each one, 
+or create a [control plane group](/konnect/gateway-manager/control-plane-groups/).
+
+Open the **Plugins** menu from within a control plane to access and manage its list of plugins:
+
+![List of plugins in a control plane](/assets/images/docs/konnect/konnect-plugin-list.png)
+
+### Custom plugins
+
+You can also [apply your own custom plugins](/konnect/gateway-manager/plugins/add-custom-plugin/) 
+through the Gateway Manager.
+
+All that the {{site.konnect_short_name}} control plane needs is a Lua schema file. Using that
+schema, it creates a plugin configuration object in {{site.konnect_short_name}}, which you
+can access via the Konnect UI or the custom plugins API. This means that {{site.konnect_short_name}}
+only sees a custom plugin's configuration options, and does not see any other data.
+
+To run in {{site.konnect_short_name}}, every custom plugin must meet the following requirements:
+
+* Admin API extensions must not contain an `api.lua` file.
+* Custom plugin database tables must not contain a `dao.lua` file.
+* The plugin must not have a `migration.lua` file.
+* The schema for your custom plugin must be written in Lua. 
+* Custom validation functions must be written in Lua and be self-contained within the schema.
+* The `schema.lua` file must not contain any `require()` statements.
+* Plugins that require third-party libraries must reference them in the `handler.lua` file.
+* Each custom plugin must have a unique name.
+
+All plugin files must also be deployed to **each** {{site.base_gateway}} data plane node.
+
+![Custom plugins](/assets/images/docs/konnect/konnect-custom-plugins.png){:.image-border}
+
+{:.important}
+> **Caution**: Carefully test the operation of any custom plugins before deploying
+them to production. Kong is not responsible for the operation or support of any 
+custom plugins, including any performance impacts on your {{site.konnect_short_name}}
+or {{site.base_gateway}} deployments. 
+
+### Application registration
+
+Application registration is built into API Products.
+[Enabling it on an API product version](/konnect/dev-portal/applications/enable-app-reg/)
+also enables two plugins in read-only mode: ACL, and one of Key Auth or OpenID
+Connect. These plugins appear in the service's plugin list, and you can view their
+configurations, but you can't edit or delete them directly.
+
+### Plugin limitations
+
+Some bundled {{site.base_gateway}} plugins are not available in {{site.konnect_short_name}}, or
+have configuration requirements specific to {{site.konnect_short_name}}.
+
+* Rate limiting plugins default to the `redis` strategy, for which you must
+provide your own Redis server. You can also use `local` to apply rate limiting
+per individual data plane.
+
+* The following plugins are not available with {{site.konnect_saas}}:
```

Comment:
```suggestion
* The following plugins are not available with {{site.konnect_saas}}:
```

Can we link to https://deploy-preview-6164--kongdocs.netlify.app/konnect/compatibility/#plugin-compatibility

instead?